sklearn/mixture/dpgmm.py

```diff
@@ -606,6 +618,8 @@ def _fit(self, X, y=None):
         return z
 
 
+@deprecated("The class VBGMM is deprecated in 0.18 and "
+            "will be removed in 0.20. This class no longer works.")
```

Comment:
Can you change the message in that way "The VBGMM class is not working correctly and it's better to not use it. DPGMM is deprecated in..."